src/main/scala-2.12/org/apache/james/gatling/jmap/JmapChecks.scala

```diff
@@ -6,6 +6,6 @@ object JmapChecks {
 
   val noError = jsonPath("$.error").notExists
 
-  def created(messageId: MessageId) = jsonPath(s"$$[0][1].created['${messageId.id}'].id").exists
```

Comment:
remove the 's' for interpolating, then you will be able to use only one '$' and remove the curly braces surrounding 'messageId'

Reply:
Just curious, what does the double `$$` stands for in a string interpolation? Cannot really find doc about it sorry

Reply:
It's to escape the `$` sign in a string interpolation (string starting by `s"`). See https://www.scala-lang.org/files/archive/spec/2.13/01-lexical-syntax.html#string-literals